designsafe/apps/api/messages/models.py

```diff
@@ -0,0 +1,21 @@
+import logging
```

Comment:
Add `import unicode_literals`: https://python-future.org/unicode_literals.html

Reply:
Importing this broke everything.

Reply:
Why did it returned a 502? What was the actual error?

Reply:
No idea, I tried to reproduce it and it seemed to have worked out okay.